SPECS-EXTENDED/libesmtp/libesmtp.spec

```diff
@@ -5,14 +5,15 @@ Distribution:   Mariner
 Summary:        SMTP client library
 Name:           libesmtp
 Version:        1.0.6
-Release:        20%{?dist}
+Release:        21%{?dist}
 License:        LGPLv2+
 #Source0:        https://github.com/libesmtp/libESMTP/archive/refs/tags/v1.0.6.tar.gz
 Source:         https://github.com/libesmtp/libESMTP/archive/refs/tags/%{name}-%{version}.tar.bz2
 URL:            https://github.com/libesmtp
 BuildRequires:  gcc
 BuildRequires:  openssl-devel pkgconfig autoconf automake libtool
 Patch0: libesmtp-1.0.6-openssl-1.1.patch
+Patch1: CVE-2019-19977-avoid-stach-overrwrite-6.patch
```

Comment:
NIT: I think we should match the spacing with the above line 14, for both line 15 & 16